modules/user-access/pages/sso.adoc

```diff
@@ -53,27 +54,56 @@ http://<tigergraph-machine-hostname>:14240/gsqlserver/gsql/saml/meta
-. Follow the Auth0 documentation to https://auth0.com/docs/configure/saml-configuration/configure-auth0-saml-identity-provider#configure-saml-sso-in-auth0[configure Auth0 as SAML Identity provider].
+. https://auth0.com/docs/get-started/create-apps/single-page-web-apps[Register TigerGraph as a Single-Page web application in Auth0].
+. https://auth0.com/docs/configure/saml-configuration/configure-auth0-saml-identity-provider#configure-saml-sso-in-auth0[Configure Auth0 as SAML Identity provider].
 You will need to enter the following information when following Auth0's guide:
 * Application Callback URL:
 [literal]
 http://<tigergraph-machine-hostname>:14240/api/auth/saml/acs`
 * When downloading the certificate from Auth0, choose *`.cer`* format instead of `.pem` format.
 
+=== Azure AD
+
+. Log into https://aad.portal.azure.com/[Azure Active Directory Admin Center] as an Admin user.
+In the left menu, select menu:Enterprise applications[New Application].
++
+image::azure-new-application.png[]
+. In the top left corner, click btn:[Create your own application].
++
+image::azure-create-app.png[]
++
+In the form that appears on the right side of the page, fill in the name of your application and choose btn:[Integrate any other application you don't find in the gallery (Non-gallery)].
+This creates the application and takes you to the application page
+. On the left pane of the application page, click btn:[Single sign-on] and choose btn:[SAML].
++
+image::azure-saml.png[]
+This takes you to a page titled "Set up single sign-on with SAML".
+. Fill in *Identifier (Entity ID)* and *Reply URL* with the information below.
+Replace `<tigergraph-machine-hostname>` with the host name of your TigerGraph instance:
+* Identifier (Entity ID): `\https://<tigergraph-machine-hostname>:14240/gsqlserver/gsql/saml/meta`
+* Reply URL: `\https://<tigergraph-machine-hostname>:14240/api/auth/saml/acs`
+. After filling in the Entity ID and Reply URL, click btn:[Edit] for *SAML Signing Certificate*, and choose btn:[Sign SAML Assertion and Response].
+Click btn:[Save] and download certificate (base64).
+* We recommend signing both assertion and response for maximum security.
+However, you may choose to only sign one of them, but you need to match this setting with when you enable SSO on TigerGraph.
+. Under "Set up <Application name>", Azure AD provides a login URL, an Azure AD identifier, and a logout URL.
+These items are used in the next steps to configure single sign-on in TigerGraph.
+
 :sectnums:
 
 == Enable Single Sign-On in TigerGraph
 
 Having configured the identity provider and obtained the identity provider's SSO URL, entity ID and certificate, the next step is to provide the information to TigerGraph to enable single sign-on.
 
+=== Navigate to SSO page
+From GraphStudio home page, click btn:[Admin Portal] in the upper right corner.
```

Comment:
The docs are inconsistent in terms of "upper-right" or "upper right." Personally I prefer "upper-right": https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/upper-left-upper-right

Reply:
I think most instances in our doc probably use upper right instead of upper-right.